### ISSUE TITLE
Add 'is-confidentiality-protected' flags-map value

### ISSUE THREAD

**Issue by nedmsmith** [ietf-rats-wg/draft-ietf-rats-corim#101]
Add new flag `is-confidentiality-protected`.

More context is needed that defines flag semantics for confidentiality protection. Presumably, it should describe "observability in memory" properties, but could also describe "encryption during supply / distribution", such as encrypted SW updates.

**Comment by yogeshbdeshpande** [comment]
@nedmsmith to come up with a better name for the flag,  and create a PR.

**Comment by nedmsmith** [comment]
The PR should add a flag to `flags-map` e.g.:
```
flags-map = {
...
? &(is-confidentiality-protected: 9) => bool
}
```
The description should be something like:
* is-tcb (index 9): If the flag is true, the measured environment is confidentiality protected. For example, if the Target Environment contains memory, sensitive information in memory is obfuscated in some way, such as encryption. If the precise confidentiality mechanism is required, a vendors-specific, or profile-specific extension can be applied. Alternatively, the model information can be used to manually obtain these details.

**Comment by nedmsmith** [comment]
Fixed by PR #116
